### PR TITLE
Fix #479 student not shown

### LINKED ISSUE #479: Students after 'Rahul Arora' not visible
**Actual Behaviour**

Even though there are names and data after Rahul Arora in "students.yml", these are not reflected on the website
**Expected Behaviour**
The students should show up on the section

**Steps to reproduce it**
Go to gci18.fossasia.org/#students

you will notice the last name to be of "Rahul Arora", even though in students.yml it's not the last name

**LogCat for the issue**

**Screenshots of the issue**

**Would you like to work on the issue?**

### PR DESCRIPTION
**Pull request by ycwei982**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://ycwei982.github.io/gci18.fossasia.org/#students)

- Fixes fossasia#479

## Description / Changes

Change `{% for student in site.data.students limit:30 %}` to `{% for student in site.data.students %}` in `index.html`.

### After:

p.s. I need to fix this before adding my profile picture to this website
- - - - - - - - - - - -

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -462,7 +462,7 @@ <h1>Students</h1>
     </div>
     <!-- end of row -->
     <div class="row equal-row">
-      {% for student in site.data.students limit:30 %}
+      {% for student in site.data.students limit:100 %}
```
Comment (abishekvashok) [inline index.html]: Don't limit students let's show them all. Remove the limit.

Comment (ycwei982) [reply index.html]: I removed the limit just now.

**Comment by jakubpawlina** [comment]
Edit the description

**Comment by ycwei982** [comment]
I edited the description and also rebased 2 commits.

**Comment by abishekvashok** [comment]
Thanks

**Pull request merged**